### DISCUSSION TITLE
Possible enhancement To: "Prefer View Mode"?

### DISCUSSION THREAD

**Discussion by nat3boi** [gsantner/markor#2156] [Q&A] [answered]
The **"Prefer View Mode"** setting is great but it seems like the files actually open in Edit mode, then quickly switch to "View Mode", rather than truly starting in View mode, thus creating a _stutter_ effect of sorts. 

This is really apparent when clicking links back and forth between markdown files. For example, I use  **_Neovim+Vimwiki_** (with markdown) on PC and sync the files to the phone and view/edit with Markor. 

It would be really cool to see a TRUE **Prefer View Mode** so that clicking links to other files is more visual pleasing.

Is this at all possible? Thank you.

--- 

_This is my first time commenting to a project that I love. I am interested in software development but I know very little about it. I have read through your page and could not find a similar discussion. Thank you for reading._

**Comment by harshad1** [comment, marked as answer]
Possibly. But IMO this is pretty low priority...

**Reply by nat3boi** [reply]
I totally understand that. Thanks for your input and contributions!
